siphon/simplewebservice/ndbc.py

```diff
@@ -0,0 +1,560 @@
+# Copyright (c) 2013-2015 Siphon Contributors.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Read data from the National Data Buoy Center."""
+
+from io import StringIO
+import warnings
+
+import numpy as np
+import pandas as pd
+import requests
+
+from ..http_util import HTTPEndPoint
+
+warnings.filterwarnings('ignore', 'Pandas doesn\'t allow columns to be created', UserWarning)
+
+
+class NDBC(HTTPEndPoint):
+    """Download and parse data from the National Data Buoy Center."""
+
+    def __init__(self):
+        """Set up endpoint."""
+        super(NDBC, self).__init__('https://www.ndbc.noaa.gov/')
+
+    @classmethod
+    def station_observations(cls):
```

Comment:
Um, this method is empty.

Reply:
Left over from spelling internal debating 😄